packages/governance/src/paramManager.js

```diff
@@ -0,0 +1,133 @@
+// @ts-check
+
+import { assert, details as X } from '@agoric/assert';
+import { assertIsRatio } from '@agoric/zoe/src/contractSupport';
+import { AmountMath, looksLikeBrand } from '@agoric/ertp';
+import { Far } from '@agoric/marshal';
+import { assertKeywordName } from '@agoric/zoe/src/cleanProposal';
+
+/**
+ * @type {{
+ *  AMOUNT: 'amount',
+ *  UNKNOWN: 'unknown',
+ *  BRAND: 'brand',
+ *  INSTANCE: 'instance',
+ *  INSTALLATION: 'installation',
+ *  NAT: 'nat',
+ *  RATIO: 'ratio',
+ *  STRING: 'string',
+ * }}
+ */
+const ParamType = {
+  AMOUNT: 'amount',
+  BRAND: 'brand',
+  INSTANCE: 'instance',
+  INSTALLATION: 'installation',
+  NAT: 'nat',
+  RATIO: 'ratio',
+  STRING: 'string',
+  UNKNOWN: 'unknown',
+};
+harden(ParamType);
+
+const assertType = (type, value, name) => {
+  switch (type) {
+    case ParamType.AMOUNT:
+      // It would be nice to have a clean way to assert something is an amount.
+      assert(
+        AmountMath.isEqual(value, value),
```

Comment:
Is there a reason to use `isEqual`? `AmountMath.coerce` *is* the clean way to do it. We just don't have a brand to test against objectively, so the call becomes:

```suggestion
        AmountMath.coerce(value.brand, value),
```

Reply:
done